doc/cascadia/profiles.schema.json

```diff
@@ -1813,6 +1813,19 @@
           "description": "True if the Terminal should use a Mica backdrop for the window. This will apply underneath all controls (including the terminal panes and the titlebar)",
           "type": "boolean",
           "default": false
+        },
+        "rainbowFrame": {
```

Comment:
I think we should mark it as `experimental.` so we declare it as "no strings attached"-kinda support. Should we really put this into 1.18?